src/test/java/hudson/plugins/ec2/ConfigurationAsCodeTest.java

```diff
@@ -0,0 +1,107 @@
+package hudson.plugins.ec2;
+
+import java.util.List;
+
+import jenkins.model.Jenkins;
+import hudson.util.Secret;
+import hudson.model.labels.LabelAtom;
+import io.jenkins.plugins.casc.misc.ConfiguredWithCode;
+import io.jenkins.plugins.casc.misc.JenkinsConfiguredWithCodeRule;
+
+import org.junit.Rule;
+import org.junit.Test;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+public class ConfigurationAsCodeTest {
+
+    @Rule
+    public JenkinsConfiguredWithCodeRule j = new JenkinsConfiguredWithCodeRule();
+
+    @Test
+    @ConfiguredWithCode("EC2CloudEmpty.yml")
+    public void testEmptyConfig() throws Exception {
+        final AmazonEC2Cloud ec2Cloud = (AmazonEC2Cloud) Jenkins.get().getCloud("ec2-");
+        assertNotNull(ec2Cloud);
+        assertEquals(ec2Cloud.getTemplates().size(), 0);
```

Comment:
```suggestion
        assertEquals(0, ec2Cloud.getTemplates().size());
```

These assertions are usually ordered `assertEquals(expected, actual)`.